assets/src/dashboard/utils/usePagePreviewSize.js

```diff
@@ -44,15 +44,16 @@ import {
  * This means things like the unitsProvider and displayElements that we import to the Dashboard from the editor
  * It's maintaining a 2:3 aspect ratio.
```

Comment:
Is this maintaining the 2:3 ratio still?

Reply:
you're right, this is confusing now. updated for better context. hopefully we can ditch the 2:3 and fullBleedHeight here soon.